wildlifecompliance/helpers.py

```diff
@@ -78,6 +82,23 @@ def is_officer(request):
     return request.user.is_authenticated() and (belongs_to_list(
         request.user, licence_officer_groups) or request.user.is_superuser)
 
+def prefer_compliance_management(request):
+    if request.user.is_authenticated():
+        preference_qs = ComplianceManagementUserPreferences.objects.filter(email_user=request.user)
```

Comment:
not sure if get_or_create is worth using here but i would argue using .get instead of .filter()[0] is probably better (in this case, doesn't matter since it's unique for the user but i feel it's better practice)